.github/workflows/ci.yml

```diff
@@ -0,0 +1,105 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      OCI_REGISTRY: ghcr.io
+      DOCKER_IMAGE_NAME: yieldlab/prebid-server-java
+
+    steps:
+    - name: Checkout repo under /my-repo
+      uses: actions/checkout@v3
+      with:
+        fetch-depth: '0'
+        path: my-repo
+
+    - name: Extract version
+      working-directory: ./my-repo
+      run: |
+        target_tag=$(cat latest-release-version)
+        echo "TARGET_TAG=$target_tag" >> $GITHUB_ENV
+
+    - name: Check if version has already been released
+      working-directory: ./my-repo
+      run: |
```

Comment:
Could be simplified to `run: git rev-list "$TARGET_TAG" > /dev/null 2>&1 && echo "TAG_ALREADY_EXISTS=1" >> $GITHUB_ENV || exit 0`